backend_restful/view_1.py

```diff
@@ -590,19 +592,28 @@ def get(self, request):
         user.name = "Mr. Name"
         user.phone_no = "+1847439202"
         return Response(RegistrationSerializer(user).data, status=status.HTTP_201_CREATED)
-    
+
     def post(self,request):
         serializer = RegistrationSerializer(data=request.data)
         if serializer.is_valid():
             #user = User()
             #TODO store data in database
             return Response ({"Registratiion Successfull"}, status.HTTP_200_OK)
         return Response(serializer.errors,status=status.HTTP_400_BAD_REQUEST)
-    
-    
-    '''need password'''
-    pass
 
-class PublicPages(APIView):
+
     '''need password'''
     pass
+
+
+class services(APIView):
+    def get(self,request):
+        available_serializer = ServicesSerializer(AvailableServices())
+        subscribed_serializer = ServicesSerializer(AvailableServices())
+        return Response ({"AvailableServices":available_serializer.data, "SubscribedServices":subscribed_serializer.data}, status.HTTP_200_OK)
```

Comment:
https://stackoverflow.com/questions/42398477/serializing-a-list-of-objects-with-django-rest-framework